### PR TITLE
fix: new filter creation examples

### PR DESCRIPTION
**Pull request by alexandreliberato**
Fix documentation, see: https://github.com/zalando/skipper/issues/2280

### CODE REVIEW

File: docs/tutorials/development.md [hunk 1]
```diff
@@ -116,16 +116,12 @@ The simplest filter possible is, if `filters.Spec` and
...
 type myFilter struct{}
 
-func NewMyFilter() filters.Spec {
+func NewMyFilter() *myFilter {
 	return &myFilter{}
 }
 
 func (spec *myFilter) Name() string { return "myFilter" }
 
-func (spec *myFilter) CreateFilter(config []interface{}) (filters.Filter, error) {
```
Comment (AlexanderYastrebov) [inline docs/tutorials/development.md:125]: This method is a part of filters.Spec interface so it must be implemented
https://github.com/zalando/skipper/blob/399b08b3eba8f01b8ff4efd5ecccbb14fb8bd2e7/filters/filters.go#L180-L190

Comment (alexandreliberato) [reply docs/tutorials/development.md]: It is everything correct now?

Comment (AlexanderYastrebov) [reply docs/tutorials/development.md]: What is the point of this change? The title says it fixes something but it does not.

Comment (alexandreliberato) [reply docs/tutorials/development.md]: It's about this issue: https://github.com/zalando/skipper/issues/2280
If I understood correctly it was necessary to add this code to the documentation. That's right? 
If Im wrong about that, may you make it more clear to me? Thanks in advance.

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**